src/pages/SingleVideoPage.vue

```diff
@@ -1,96 +1,181 @@
 <template>
   <!-- es-lint ignore -->
-  <div class="relative py-16 sm:py-24 mx-auto px-4 max-w-6xl">
-    <div class="lg:mx-auto lg:max-w-7xl lg:px-8 lg:grid lg:grid-cols-1 lg:gap-24 lg:items-start">
+  <div v-if="authorStatus === 'ready'"  class="relative pt-16 sm:pt-24 mx-auto px-4 max-w-6xl ">
+    <div
+      class="
+        lg:mx-auto
+        lg:max-w-7xl
+        lg:px-8
+        lg:grid lg:grid-cols-1
+        lg:gap-24
+        lg:items-start
+      "
+    >
```

Comment:
a lot of lines got formatted by prettier, can we stick to just keeping them in a single line?

Reply:
I think we should be styling everything with prettier so everything will be the same. But we need to define the rules we want in a config file so it will be the same.

Reply:
```json
"vetur.validation.template": false,
"prettier.printWidth": 1000,
"prettier.useEditorConfig": false,
"prettier.arrowParens": "avoid",
"prettier.trailingComma": "none",
```

This should fix the prettier issue as we discussed earlier right?

Reply:
I did not change anything since the last time we worked on it together. some files are using single quotes others are using double-quotes.

Reply:
> ```json
> "vetur.validation.template": false,
> "prettier.printWidth": 1000,
> "prettier.useEditorConfig": false,
> "prettier.arrowParens": "avoid",
> "prettier.trailingComma": "none",
> ```
> 
> This should fix the prettier issue as we discussed earlier right?

This is for the formatting actually. If you have this on your settings, the code will be single line. Maybe you have another config that overwrites this config. 

For the single quotes, there is another setting on vscode that you can change to make quotes as single quotes. We did it on a quick call together on my computer if I'm not mistaken.